mk/image.lds.S

```diff
@@ -92,15 +94,23 @@ SECTIONS {
 
 		ALIGNMENT();
 		_reserve_end = .;
+		_bss_reserve = _reserve_end - _bss_end;
+
 	} SECTION_REGION(bss) :bss
 
 	_bss_len = _bss_end - _bss_vma;
 	_bss_len_with_reserve = SIZEOF(.bss);
 
+	_heap_vma = 0;
+	_heap_end = 0;
+	_heap_len = 0;
 #if HAS_SECTION_REGION(heap)
 	.heap : ALIGN(DATA_ALIGNMENT) {
+		_heap_vma = .;
```

Comment:
Unfortunately, as I can see, the section .heap is completely unused. The actual heap is just a part of the .bss..reserve section and can be found here src/mem/heap/heap.lds.S There also HEAP_START and HEAP_END are defined, so you can use these instead on head_vma etc.

Reply:
It's my mistake, this section is used for static heap on stm32.

Reply:
I can't understand, how I can use HEAP_START and HEAP_END variables, becouse they defined by macros OPTION_GET(STRING,start_symbol) and I don't know real name of this variables.
Should I add knew variables, describing heap start and end, or exist a way to solve this problem better?

Reply:
You can use in your file something like this
```
#define HEAP_START OPTION_MODULE_GET(embox__mem__static_heap,STRING,start_symbol)
extern char HEAP_START;
```
But it is only for one heap implementation (static_heap).

I'd propose to leave this as is and print only text, data, rodata, and bss. The problem with heap  could be deferred for the future :)

Reply:
Ok, I removed all information about heap.